libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -166,10 +170,14 @@ benchmark bench
     QuickCheck,
     cardano-ledger-shelley,
     cardano-ledger-shelley-test,
+    cardano-slotting,
     microlens,
+    mtl,
     random,
     small-steps,
     small-steps-test,
+    strings,
```

Comment:
This package is really outdated and is unneeded

Reply:
removed